### DISCUSSION TITLE
RAF not working for me?

### DISCUSSION THREAD

**Discussion by chunkthefat** [chromiecraft/chromiecraft#2321] [General]
I did Recruit a friend for 2 of my friends. They both now say removed/expired? I play with them everyday. I was recruited by someone and it is still working with that but not with my family.

**Comment by 55Honey** [comment]
Hi chunkthefat,
Please read the rules regarding RAF: Same-IP recruitment isn't possible. The this site: https://www.chromiecraft.com/recruit-a-friend-is-available-now/
That's why it was removed automatically.
